### PR TITLE
Fixed typos and grammar issues

### PR DESCRIPTION
**Pull request by rziehl**

### CODE REVIEW

File: data/v1/subjects/javascript/courseList.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   },
   {
     "id": "javascript-comments",
-    "name": "Comment",
+    "name": "Comments",
```
Comment (rziehl) [inline data/v1/subjects/javascript/courseList.json:18]: This one wasn't pluralized

File: data/v1/subjects/javascript/courseList.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
   },
   {
     "id": "javascript-undefined",
-    "name": "No values?",
+    "name": "No Values?",
```
Comment (rziehl) [inline data/v1/subjects/javascript/courseList.json:53]: Fixed casing for consistency with other titles

File: data/v1/subjects/javascript/courses/undefined.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
   {
     "type": "outline",
     "title": "No value assigned",
-    "content": "When we learn how to create variables, we always had a value assigned to it. You can also just create variables with no values assigned to it. In the code below myFirstVariable is assigned to a default value called undefined. This always happens when any variable is created and doesn't have anything assigned to it, otherwise to equal sign after the variable name.",
+    "content": "When we learn how to create variables, we always had a value assigned to it. You can also just create variables with no values assigned to it. In the code below myFirstVariable is assigned to a default value called undefined. This always happens when any variable is created and doesn't have anything assigned to it.",
```
Comment (rziehl) [inline data/v1/subjects/javascript/courses/undefined.json:13]: Wasn't sure what to do here. The end of this sentence makes no sense.

**Comment by awaseem** [comment]
You are a gem ❤️

**Pull request merged**